source/server/config/access_log/grpc_access_log.cc

```diff
@@ -41,14 +41,20 @@ AccessLog::InstanceSharedPtr HttpGrpcAccessLogFactory::createAccessLogInstance(
   const auto& proto_config = MessageUtil::downcastAndValidate<
       const envoy::api::v2::filter::accesslog::HttpGrpcAccessLogConfig&>(config);
 
+  // TODO(htuch): Support Google gRPC client.
+  const auto cluster_name = proto_config.common_config().grpc_service().envoy_grpc().cluster_name();
+  auto cluster = context.clusterManager().get(cluster_name);
+  if (cluster == nullptr || cluster->info()->addedViaApi()) {
```

Comment:
what is the difference between a static cluster vs a cluster added by api?

Reply:
Static cluster is one defined in the config. A cluster added by api would be in the CDS case.